mt2.tex

```diff
@@ -18,4 +22,373 @@
+	Continuing from the previous slide we have: \\
+	\[
+	\begin{bmatrix} f_1(\vec{x}) \\ \vdots \\f_n(\vec{x}) \end{bmatrix} \approx 
+	\begin{bmatrix} f_1(\vec{x}^*) \\ \vdots \\f_n(\vec{x}^*) \end{bmatrix} + 
+	\begin{bmatrix} \frac{\partial f_1}{\partial x_1}(\vec{x}^*)& \hdots& \frac{\partial f_1}{\partial x_n}(\vec{x}^*) \\ \vdots & \ddots &\vdots \\ \frac{\partial f_n}{\partial x_1}(\vec{x}^*)& \hdots &  \frac{\partial f_n}{\partial x_n}(\vec{x}^*) \end{bmatrix}
+	(\vec{x} - \vec{x}^*)
+	\]
+	\\
+
+	\end{frame}
+
+	\begin{frame}{Linearization with Jacobians Example}
+	    \begin{align*}
+	    \text{Linearize}  \hspace{0.2 cm} 
+	     \vec{f}(\vec{x}(t))  = 
+	    \begin{bmatrix}
+	    sin(x_1(t)*x_2(t)) + 2x_1(t)x_3^2(t)\\
+	    x_3(t)cos(x_2(t)) + \frac{x_1(t)}{x_3(t)}\\
+	    x_1(t) + 2x_3(t)x_2^3(t)
+	    \end{bmatrix} 
+	    \text{about} \hspace{0.2 cm} \vec{x}^* = \begin{bmatrix}
+	    0\\ 2 \pi \\ \frac{2 \pi}{3}
+	    \end{bmatrix} 
+	    \end{align*}
+	\end{frame}
+
+	\begin{frame}{Solutions}
+	 Find the Jacobian: 
+	\small
+	\[
+	        \begin{bmatrix}
+	        x_2(t)cos(x_1(t)*x_2(t)) + 2x_3^2(t)& x_1(t)cos(x_1(t)*x_2(t)) & 4x_1(t)x_3(t)\\ 
+	        \frac{1}{x_3(t)}& -x_3(t)sin(x_2(t)) &cos(x_2(t)) - \frac{x_1(t)}{x_3^2(t)}  \\ 
+	        1 & 6x_3(t)x_2^2(t)& 2 x_2^3(t) 
+	        \end{bmatrix}
+	\]
+	\normalsize
+	Evaluate the Jacobian about $\vec{x}^*$:
+	\[
+	        \begin{bmatrix}
+	       5 \pi & 0 & 0\\
+	       \frac{2 \pi }{3}& 0 & 1\\
+	       1 & 36 \pi^3 & 16 \pi^3
+	        \end{bmatrix}
+	\]
+	\\
+	Linearize:
+	\begin{equation*}
+	 \vec{f}(\vec{x}(t)) \approx 
+	 \begin{bmatrix}
+	 0 \\ \frac{3\pi }{4} \\ 24 \pi^4
+	 \end{bmatrix} +
+	 \begin{bmatrix}
+	       5 \pi & 0 & 0\\
+	       \frac{2 \pi }{3}& 0 & 1\\
+	       1 & 36 \pi^3 & 16 \pi^3
+	 \end{bmatrix}
+	 \begin{bmatrix}
+	 x_1(t) - 0 \\ x_2(t) - \frac{3\pi }{4} \\ x_3(t) - 24 \pi^4
+	 \end{bmatrix}
+	\end{equation*}
+
+	\end{frame}
+
+	\begin{frame}{Steps to Linearize Vector ODE Systems}
+	To linearize $\frac{d\vec{x}}{dt} = \vec{f}(\vec{x}(t), \vec{u}(t))$ we use a similar procedure as we did for the scalar case. \pause 
+	\begin{enumerate}[(i)]
+	    \item If you're not given a DC input $\vec{u}^*$, determine one. \pause \\
+	    \item Solve $\vec{f}(\vec{x}^*, \vec{u}^*) = \vec{0}$ to determine the DC operating point. \pause \\
+	    \item Let $\vec{x}_l(t) = \vec{x}(t) - \vec{x}^*$ and $\vec{u}_l(t) = \vec{u}(t) - \vec{u}^*$ \pause \\
+	    \item Linearize $\vec{f}(\vec{x}, \vec{u})$ about $(\vec{x}^*, \vec{u}^*)$. That is: $\vec{f}(\vec{x}(t), \vec{u}(t)) \approx \vec{f}(\vec{x}^*, \vec{u}^*) + J_{\vec{x}}\vec{x}_l(t) + J_{\vec{u}}\vec{u}_l(t)$ \pause \\
+	    \item Plug (iv) back into the ODE: $\frac{d\vec{x}}{dt} \approx \cNCEL{\vec{f}(\vec{x}^*, \vec{u}^*)} + J_{\vec{x}}\vec{x}_l(t) + J_{\vec{u}}\vec{u}_l(t)$
```

Comment:
What is cNCEL ? It's not compiling on my environment

Reply:
should be \cancel sorry!